### PR TITLE
Add an example for local mode deployment of models

### PR DESCRIPTION
**Pull request by iquintero**
If there is an existing model it can be deployed locally
using local mode. This feature is already present but there
is no example in the README.

*Issue #, if available:*

*Description of changes:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -192,6 +192,30 @@ instance type.
     mxnet_estimator.delete_endpoint()
 
 
+If you have an existing model and would like to deploy it locally you can do that as well. If you don't
+specify a sagemaker_session argument to the MXNetModel constructor, the right session will be generated
+when calling model.deploy()
```
Comment (laurenyu) [inline README.rst]: This still sounds like a choice to me - shouldn't we just instruct the user to not supply a session object? (or would there be a reason for a user to want to construct a local session separately?)

Comment (iquintero) [reply README.rst]: this is similar to how the rest of the SDK works. they can provide their own Local Session if they want, with custom config settings.

Comment (iquintero) [reply README.rst]: what is a better wording ?

Comment (laurenyu) [reply README.rst]: maybe something like "This requires a Local Session (maybe include a link to documentation?) to run. If you don't specify..." or "If you want to specify a sagemaker session, make sure it's a local session"

**Pull request merged**